packages/card/package.json

```diff
@@ -23,8 +23,10 @@
         "./src/card.css.js": "./src/card.css.js"
     },
     "dependencies": {
-        "@swc-uxp-internal/card": "npm:@spectrum-web-components/card@0.13.4",
-        "@swc-uxp-wrappers/checkbox": "1.0.0"
+        "@swc-uxp-internal/card": "npm:@spectrum-web-components/card@0.37.0",
+        "@swc-uxp-wrappers/checkbox": "1.0.0",
```

Comment:
Version of elementary wrappers must be updated (2.0.0)

Reply:
As discussed earlier, this version does not yet. So lets not put it right now.
once we bump up the package versions during that stage we should update the transitive dependencies as well.